onnxruntime/core/optimizer/qdq_transformer/selectors_actions/qdq_actions.cc

```diff
@@ -94,8 +94,35 @@ struct SetOptionalZeroPoint {
   static void UpdateNodes(Graph&, const NodesToOptimize& selected_nodes);
 
  private:
-  static const ONNX_NAMESPACE::TensorProto optional_zero_point_int8_;
-  static const ONNX_NAMESPACE::TensorProto optional_zero_point_uint8_;
+  //We assume this function won't fail
+  static const ONNX_NAMESPACE::TensorProto init_optional_zero_point_int8() {
+    const char* const name = "b33fd0fa-cd7b-4b10-ae5a-df64cabfe1f8";  // guid as arbitrary name to provide a unique value
```

Comment:
>b33fd0fa [](http://example.com/codeflow?start=30&length=8)

Amazing. Perhaps, we could prefix it with some human readable thing, such as init_optional_XXXX. However, theze TensorProtos will all have the same name?